lib/meson.build

```diff
@@ -277,9 +279,15 @@ foreach l : all_libs
                 set_variable('te_lib_' + libname, lib)
                 lib_static = lib
 
-                dep = declare_dependency(link_with: lib,
-                                         include_directories: includes,
-                                         dependencies: deps_static)
+                if link_whole
+                    dep = declare_dependency(link_whole: lib,
+                                             include_directories: includes,
+                                             dependencies: deps_static)
```

Comment:
It looks like a dead code in the patch. I see no way to set link_whole==true. Am I missing something?